src/braze/tags/connectedContent.ts

```diff
@@ -18,11 +18,11 @@ export default <ITagImplOptions>{
     const options = match[2]
     this.options = {}
     if (options) {
-      options.split(/\s*:/).forEach((optStr) => {
+      options.split(/\s+:/).forEach((optStr) => {
         if (optStr === '') return
 
         const opts = optStr.split(/\s+/)
-        this.options[opts[0]] = opts.length > 1 ? opts[1] : true
+        this.options[opts[0]] = opts.length > 1 ? opts.slice(1).join(" ") : true;
```

Comment:
This doesn't seem to work if there's a space before `:` in headers, but it's a valid JSON format
```
{ "testHeader" : "headerValue" } 
```

maybe we should have another regex to extract `:headers\s+\{.*?\}` before parsing options so this section can be unchanged?

Reply:
I found a workaround. I had to modify that slightly for multi-line json.